test/functional/wallet_multiwallet.py

```diff
@@ -109,7 +115,16 @@ def wallet_file(name):
             self.nodes[0].createwallet(wallet_name, descriptors=False)
         for wallet_name in wallet_names[-2:]:
             self.nodes[0].loadwallet(wallet_name)
-        assert_equal(sorted(map(lambda w: w['name'], self.nodes[0].listwalletdir()['wallets'])), [self.default_wallet_name, os.path.join('sub', 'w5'), 'w', 'w1', 'w2', 'w3', 'w7', 'w7_symlink', 'w8'])
+
+        os.mkdir(wallet_dir('no_access'))
+        os.chmod(wallet_dir('no_access'), 0)
```

Comment:
44c0bc31f4fadc932877d9764402644c84f3be4c

If this line is commented out, the test still passing.

Reply:
???

Reply:
If initial state is changed, i.e., the `no_access` directory actually has access, the test should fail, right?

Reply:
No, it's just there to make sure it doesn't break things. We're testing bitcoind, not the OS...

(And in fact, chmod won't work to deny read access on Windows)